app/views/instruments/schedule.html.haml

```diff
@@ -40,7 +40,7 @@
   %p.notice= t(".no_upcoming_admin_reservations")
 
 - else
-  - unless @instrument.online?
+  - if @instrument.offline?
```

Comment:
I think I might have been thinking this was a different view. It could be a different method, but there should probably be some warning here if it's offline via the shared schedule.